### PR TITLE
Ques-ans-page-frontend

### PR DESCRIPTION
**Pull request by mayank-del**
## Issue that this pull request solves

#700

## Types of changes

_Put an `x` in the boxes that apply_

- [ X] New feature (non-breaking change which adds functionality)

## Checklist

_Put an `x` in the boxes that apply_

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] My changes generate no new warnings
- [ x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Q A_page

### CODE REVIEW

File: frontend/src/pages/Q&A/Q&A.jsx [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+            <div
+              className={
+                dark
+                  ? `${style["resource-input"]} ${style["resource-input-dark"]} `
+                  : `${style["resource-input"]} ${style["resource-input-light"]}`
+              }
+            >
+              <input
+                autoFocus="on"
+                placeholder="Title"
```
Comment (AyushSingh22) [inline frontend/src/pages/Q&A/Q&A.jsx:163]: You ask about the subject instead of Title of the question.

**Review by AyushSingh22 — CHANGES_REQUESTED** [review]
Work on the remarks i have commented Rest is fine, Keep developing

File: frontend/src/pages/Q&A/Q&A.jsx [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+                dark
+                  ? `${style["resource-input"]} ${style["resource-input-dark"]} `
+                  : `${style["resource-input"]} ${style["resource-input-light"]}`
+              }
+            >
+              <input
+                placeholder="Body"
+  
+                type="text"
+                name="body"
```
[the remark this answers is not in the document]
Comment (AyushSingh22) [reply frontend/src/pages/Q&A/Q&A.jsx]: Yes, it's better. Please place the image with uniform padding, the image is slightly to the left.

**Comment by mayank-del** [comment]
Very sorry @AyushSingh22 , actually i forgot two things-:
1st styling
2nd making it popup 

I will do both asap

**Comment by mayank-del** [comment]
now see @AyushSingh22  ,is it ok?

GIF:

hitk_gif

File: frontend/src/pages/Resources/components/ResourceSharingForm/resource-sharing-form.module.scss [hunk 3]
```diff
@@ -301,9 +334,17 @@ input[type="date"]:not(:valid):before {
 .submit-btn {
   display: flex;
   justify-content: center;
+
   width: 40%;
   margin: 0 auto;
 }
+.Ask-div-main {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Resources/components/ResourceSharingForm/resource-sharing-form.module.scss:341]: Please format this file - each style should be separated by an empty line.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
By "pop up", I meant something like -
https://images.app.goo.gl/nbCLWJbGD3RDetCG8

Can we please have it? @mayank-del

File: frontend/src/pages/Resources/components/ResourceSharingForm/resource-sharing-form.module.scss (hunk 3 above)
Comment (mayank-del) [reply frontend/src/pages/Resources/components/ResourceSharingForm/resource-sharing-form.module.scss:341]: Ok i will

**Comment by mayank-del** [comment]
> By "pop up", I meant something like - https://images.app.goo.gl/nbCLWJbGD3RDetCG8
> 
> Can we please have it? @mayank-del

Ok

**Comment by mayank-del** [comment]
@Kajol-Kumari , i made a popup like https://images.app.goo.gl/nbCLWJbGD3RDetCG8

**Comment by mayank-del** [comment]
final popup is  pushed, please check

**Comment by Divijkatyal0406** [comment]
> final popup is pushed, please check

Please review the responsiveness. Not responsive for frame size lower than 500px

**Comment by mayank-del** [comment]
> > final popup is pushed, please check
> 
> Please review the responsiveness. Not responsive for frame size lower than 500px

Except that, everything is ok???

**Comment by Divijkatyal0406** [comment]
> > > final popup is pushed, please check
> > 
> > 
> > Please review the responsiveness. Not responsive for frame size lower than 500px
> 
> Except that, everything is ok???

Yes just review the responsiveness

**Comment by mayank-del** [comment]
> > > > final popup is pushed, please check
> > > 
> > > 
> > > Please review the responsiveness. Not responsive for frame size lower than 500px
> > 
> > 
> > Except that, everything is ok???
> 
> Yes just review the responsiveness

please check it carefully, my page is responsive upto 300px, just see the screenshot.

new

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@mayank-del Few corrections required-
1. Please remove the image from `Q&A` page and just keep the `Ask Question` card in the centre of the page.
2. After you remove the image, please make sure that footer remains sticked to th ebotton of the page
3. The pop is looking really good but for the light mode I am not able to view the checkbox data. Can you please fix that?
Screenshot 2022-03-31 at 11 28 53 AM

**Comment by mayank-del** [comment]
> @mayank-del Few corrections required-
> 
> 1. Please remove the image from `Q&A` page and just keep the `Ask Question` card in the centre of the page.
> 2. After you remove the image, please make sure that footer remains sticked to th ebotton of the page
> 3. The pop is looking really good but for the light mode I am not able to view the checkbox data. Can you please fix that?
> 
> Screenshot 2022-03-31 at 11 28 53 AM

now ,check it, I think I have made all required changes

**Comment by mayank-del** [comment]
@Kajol-Kumari  , please check , I think I have made all required changes

File: frontend/src/pages/Q&A/Ques.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  background: #171717;
+}
+
+.question_form {
+  width: 50%;
+  border: 2px solid #2a4886;
+  border-radius: 7px;
+  margin: 5% 5% 5% 25%;
+}
+.close-popup {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Q&A/Ques.scss:13]: please format this file. Each style should be a single blank space separated.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@mayank-del minor left-out changes -
1. Can we please have the background-color as black only for dark mode as for light mode it's not looking good
Screenshot 2022-04-03 at 11 09 51 AM

2. Can we please get rid of `Q&A.svg` as we don't need it now ig?

Rest Looks good to go :)

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for making the requested changes @mayank-del 
LGTM 👍

**Comment by mayank-del** [comment]
> Thanks for making the requested changes @mayank-del LGTM 👍

It's my pleasure 🙏🙏🙏

**Pull request merged**